packages/mc-scripts/config/create-webpack-config-for-production.js

```diff
@@ -160,8 +160,10 @@ module.exports = ({
     },
 
     plugins: [
-      new CleanWebpackPlugin([distPath], {
-        allowExternal: true,
+      new CleanWebpackPlugin({
+        dangerouslyAllowCleanPatternsOutsideProject: true,
+        dry: true,
+        cleanOnceBeforeBuildPatterns: [distPath],
```

Comment:
This now hooks into the webpack config and normally doesn't need the dist path. However in this case we do need it as we seem to clean outside of the project.